tests/testthat/test-rand_forest-aorsf.R

```diff
@@ -29,6 +29,42 @@ test_that("model object", {
   )
 })
 
+# prediction: time --------------------------------------------------------
+
+test_that("time predictions", {
+  skip_if_not_installed("aorsf")
```

Comment:
turns out that the suggests version requirement is a little more loose than imports. So I was able to get into a situation where this was needed.

```suggestion
  skip_if_not_installed("aorsf", "0.1.2")
```

Inn general it doesn't matter much, but I also think it is good practice in this case

Reply:
Thanks!